sickbeard/providers/rarbg.py

```diff
@@ -136,7 +134,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                 # Don't log when {"error":"No results found","error_code":20}
                 # List of errors: https://github.com/rarbg/torrentapi/issues/1#issuecomment-114763312
                 if error:
-                    if try_int(error_code) != 20:
+                    if try_int(error_code) != 20 or try_int(error_code) != 14:
```

Comment:
Ok Let me rephrase this.. You want to raise a warning for All error codes, but not 20 and 14? In that case just do a: not in (20, 14)